float64[] com_position
float64[] com_velocity
